solr/test-framework/src/java/org/apache/solr/util/SolrJettyTestRule.java

```diff
@@ -94,33 +102,37 @@ public void startSolr(Path solrHome) {
 
   public void startSolr(Path solrHome, Properties nodeProperties, JettyConfig jettyConfig) {
 
-    jetty = new JettySolrRunner(solrHome.toString(), nodeProperties, jettyConfig);
-    try {
-      jetty.start();
-    } catch (RuntimeException e) {
-      throw e;
-    } catch (Exception e) {
-      throw new RuntimeException(e);
+    if (jetty == null) {
+
+      jetty = new JettySolrRunner(solrHome.toString(), nodeProperties, jettyConfig);
+      try {
+        jetty.start();
+      } catch (RuntimeException e) {
+        throw e;
+      } catch (Exception e) {
+        throw new RuntimeException(e);
+      }
+      int port = jetty.getLocalPort();
+      log.info("Jetty Assigned Port#{}", port);
+      adminClient = getHttpSolrClient(jetty.getBaseUrl().toString());
     }
-    int port = jetty.getLocalPort();
-    log.info("Jetty Assigned Port#{}", port);
-    adminClient = getHttpSolrClient(jetty.getBaseUrl().toString());
   }
 
   public JettySolrRunner getJetty() {
     if (jetty == null) throw new IllegalStateException("Jetty has not started");
     return jetty;
   }
 
+  public SolrClient getClient() {
```

Comment:
What's this for?  Seems redundant with getSolrClient; no?  If you had documented it, maybe you would have self-discovered the duplicity